### DISCUSSION TITLE
Request for Refund due to Not Working Properly

### DISCUSSION THREAD

**Discussion by HelloWorld6493** [aclap-dev/video-downloadhelper#388] [License & Purchase requests]
### What's your issue?

Dear Video DownloadHelper team,

I am unable to download any video on my Google Chrome browser and I would like to request for a refund. 

Please let me know what information your team needs to process me a refund.

Thank you😊

### The 10 first digits (only 10) of your license number or email address

christopherhywang@gmail.com

### I confirm that I did not not paste my full license number

- [x] I confirm

**Comment by jcfrog** [comment]
Hi,
You have been refunded. Should be available in a few business days.
Regards,
jerome
